.github/dependabot.yml

```diff
@@ -4,3 +4,5 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    commit-message:
+      prefix: "ci"
```

Comment:
I think dependency changes should belong to `chore`, aka current value. `ci` should refer to changes to CI configuration (like this PR), not modifications made by CI.

Reply:
Currently I see depandabot PRs for github-actions only. I suppose this is because we don't have a lockfile.

Reply:
Maybe we can try [renovate]()https://docs.renovatebot.com/?

Reply:
We are exploring it - https://github.com/eslint/eslint/issues/15726

Reply:
I agree with @kecrily. “ci” doesn’t seem appropriate for this type of change.

Reply:
From https://eslint.org/docs/latest/developer-guide/contributing/pull-requests `ci - changes to our CI configuration files and scripts.`.
The dependabot updates to GitHub Actions, are only to CI scripts in the `.github/workflows` folder. If Dependabot is later used for other types of updates like NPM, the default of `chore` would still be used, even with this change

Reply:
Ah fair point! I didn't realize this was scoped to just GitHub Actions. In that case, I agree that `ci` makes sense. :+1: